src/components/personalinformation/PersonalInfoItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, ViewHidden } from '~/assets/image';
+
+const PersonalInfoItem = ({ label, value, visibility, bold, preview }) => {
+  return (
+    <div className="personal-info-row">
+      <div className="personal-info-row__item">
+        <div className="personal-info-row__label">Your {label.charAt(0).toUpperCase() + label.slice(1)}</div>
```

Comment:
May I ask what kind of string are we expecting for label value. 
If you are uppercasing the first char of label, you can use: 
1. `text-transform: capitalize` for a single word.
2.  ::first-letter { text-transform: uppercase; } for more then one word.

Reply:
I have updated that, will push my changes